# true when requested a picture and false after the picture is taken
bool taken_photo
# true when starting a video and false when video is stopped
bool taken_video
# true when starting a video and false when video is stopped
int16 N_photo_taken
# true when starting a video and false when video is stopped
int16 N_video_taken